apps/aecore/src/aec_block_fork.erl

```diff
@@ -0,0 +1,25 @@
+-module(aec_block_fork).
+
+-export([apply_minerva/1]).
+
+-spec apply_minerva(aec_trees:trees()) -> aec_trees:trees().
+apply_minerva(Trees) ->
+    Accounts = aec_fork_block_settings:minerva_accounts(), 
+    AccTrees =
+        lists:foldl(
+            fun({Pubkey, Balance}, AccumAccTrees) when
+                  is_integer(Balance) andalso Balance > 0
```

Comment:
In genesis a [balance was 0](https://github.com/aeternity/aeternity/blob/feba9b6811df5b910360c669967466d3045e96c5/data/aecore/.genesis/accounts.json#L124). So no balance 0 for minerva hard fork accounts file?

Reply:
Lets try to remember this...